economy/schema.py

```diff
@@ -217,16 +255,40 @@ def get_node(cls, info, id):
         return SociOrderSession.objects.get(pk=id)
 
 
+class SociProductNodeWithDefault(DjangoObjectType):
+    class Meta:
+        model = SociProduct
+        interfaces = (Node,)
+        include_fields = ("is_default",)
+
+    is_default = graphene.Boolean()
+
+
 class SociProductQuery(graphene.ObjectType):
     soci_product = Node.Field(SociProductNode)
     all_active_soci_products = DjangoConnectionField(SociProductNode)
     all_soci_products = graphene.List(SociProductNode)
+    all_soci_products_with_default = graphene.List(SociProductNodeWithDefault)
```

Comment:
This can probably be done by using the `default_stilletime_product` field on the existing node and use this filter value on the frontend.